modules/t/archiveStableId.t

```diff
@@ -227,6 +229,21 @@ $archive_obj = $asia->fetch_by_stable_id('ENSG00000171456.1');
 is($archive_obj->stable_id, 'ENSG00000171456', 'fetch_by_stable_id with active stable_id with version');
 ok($archive_obj->is_latest, 'Is the latest stable_id');
 
+# Test querying with a multispecies DBA
+{
+  $asia->is_multispecies(1);
+  $asia->species_id(2);
+  is(@{$asia->list_mapping_session()}, 0, 'Now a multi-species DBA of ID 2 so no mapping_session should be expected');
+  is(@{$asia->list_dbIDs()}, 0, 'Now a multi-species DBA of ID 2 so no mapping_session should be expected');
+  $asia->is_multispecies(0);
+  $asia->species_id(1);
+}
```

Comment:
testing unnecessary methods
and none of the modified methods are actually tested for multi-species support